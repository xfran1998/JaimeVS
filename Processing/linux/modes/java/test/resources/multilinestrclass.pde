class TestClass {
  final String testMultiline1 = """
line1 "
line 2 ""
line  3
line   4""";

  String getStr() {
    return testMultiline1;
  }
}


void setup() {
  TestClass test = new TestClass();
  println(test.getStr());
}

void draw() {
}
