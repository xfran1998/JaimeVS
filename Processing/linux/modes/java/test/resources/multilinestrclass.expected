import processing.core.*;
import processing.data.*;
import processing.event.*;
import processing.opengl.*;

import java.util.HashMap;
import java.util.ArrayList;
import java.io.File;
import java.io.BufferedReader;
import java.io.PrintWriter;
import java.io.InputStream;
import java.io.OutputStream;
import java.io.IOException;

public class multilinestrclass extends PApplet {

class TestClass {
  final String testMultiline1 = "\nline1 \"\nline 2 \"\"\nline  3\nline   4";

   public String getStr() {
    return testMultiline1;
  }
}


 public void setup() {
  TestClass test = new TestClass();
  println(test.getStr());
}

 public void draw() {
}


    static public void main(String[] passedArgs) {
        String[] appletArgs = new String[] { "multilinestrclass" };
        if (passedArgs != null) {
            PApplet.main(concat(appletArgs, passedArgs));
        } else {
            PApplet.main(appletArgs);
        }
    }
}
